File: BuilderDesignPattern/Email_Example/Director/Director.java
package BuilderDesignPattern.Email_Example.Director;

import BuilderDesignPattern.Email_Example.Products.Email;

public class Director {
	public Email outOfOffice(String recipient) {
		return new Email.Builder(recipient, "Out of Office till 31st March 2025")
					   .addCCRecipient("aadityakasbekar2@gmail.com", "kartik28@gmail.com")
					   .setBody("Hi,\nI am Aaditya currently out of office till 31st March. If you require any urgent assistance or support my team member Kartik Raut will be able to help.\nHis email is kartik28@gmail.com.\nSee you soon. Thank you.\n\nKind Regards\nAaditya Kasbekar\n")
					   .build();
	}
}

File: BuilderDesignPattern/Email_Example_Python/Director/Director.py
from BuilderDesignPattern.Email_Example_Python.Builders.EmailBuilder import EmailBuilder
from BuilderDesignPattern.Email_Example_Python.Products.Email import Email


class Director:

	def outOfOfficeEmail(self, latestSender: str) -> Email:
		return (EmailBuilder(latestSender, "Out of Office till 30th March.")
				.addCCRecipients("aadityakasbekar2@gmail", "kartikraut98@gmail.com")
				.addBody(f"""
				Hi,

				Thanks for reaching out. I hope you are doing well. Currently, I am out of office till 30th March 2026.
				I will get back to you as soon as possible after my return. 

				If this issue requires urgent attention, please contact my teammate Kartik Raut (kartikraut98@gmail.com).

				Thanks & Regards,
				Aaditya Kasbekar
				""")
				.build())
